environments/aks/demo.tfvars

```
@@ -14,12 +14,19 @@ system_node_pool = {
   min_nodes = 4,
   max_nodes = 10
 }
+
 linux_node_pool = {
   vm_size   = "Standard_D8ds_v5",
   min_nodes = 20,
   max_nodes = 30,
   max_pods  = 50
 }
 
+spot_node_pool = {
+  vm_size   = "Standard_D8ds_v5",
+  min_nodes = 1,
+  max_nodes = 10
```

shouldn't this match linux? although I'd expect it to decrease while we are using spot
```suggestion
  max_nodes = 30
```

maybe 15?
```suggestion
  max_nodes = 15
```